.github/workflows/purge-test-resources.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
           account-type: org
           org-name: radius-project
           token: ${{ secrets.GH_RAD_CI_BOT_PAT }}
+          keep-at-least: 1
```

Comment:
..so they don't allow force deletion of all versions?

How does `keep-at-least: 1` decide if more than one should be kept? Would using `keep: 1` be more appropriate for our scenario since we don't have a reason to keep more packages around?

Reply:
Looks like that, since initial bunch for errors are all similar to 
```
Couldn't delete dev/test/functional-portable/corerp/recipes/rabbitmq-recipe:185826127.
[168](https://github.com/radius-project/radius/actions/runs/9112471250/job/25051787578#step:2:173)Status code: 400
[169](https://github.com/radius-project/radius/actions/runs/9112471250/job/25051787578#step:2:174)Response: {'message': 'You cannot delete the last tagged version of a package. You must delete the package instead.', 'documentation_url': 'https://docs.github.com/rest/packages/packages#delete-package-version-for-an-organization'}Couldn't delete dev/test/functional-portable/corerp/recipes/rabbitmq-recipe:185826127.
[168](https://github.com/radius-project/radius/actions/runs/9112471250/job/25051787578#step:2:173)Status code: 400
[169](https://github.com/radius-project/radius/actions/runs/9112471250/job/25051787578#step:2:174)Response: {'message': 'You cannot delete the last tagged version of a package. You must delete the package instead.', 'documentation_url': 'https://docs.github.com/rest/packages/packages#delete-package-version-for-an-organization'}Couldn't delete dev/test/functional-portable/corerp/recipes/rabbitmq-recipe:185826127.
[168](https://github.com/radius-project/radius/actions/runs/9112471250/job/25051787578#step:2:173)Status code: 400
[169](https://github.com/radius-project/radius/actions/runs/9112471250/job/25051787578#step:2:174)Response: {'message': 'You cannot delete the last tagged version of a package. You must delete the package instead.', 'documentation_url': 'https://docs.github.com/rest/packages/packages#delete-package-version-for-an-organization'}
```

I don't find a keep:  in their documentation.. But keep-at-least is documented as keeping atleast x number irrespective of the age of the image: https://github.com/snok/container-retention-policy#keep-at-least